src/public/app/entities/fattachment.ts

```diff
@@ -57,6 +57,18 @@ class FAttachment {
         return this.froca.notes[this.ownerId];
     }
 
+    getOwnerId() {
+      return this.ownerId;
+    }
+
+    getContentLength() {
+      return this.contentLength;
+    }
+
+    getUtcDateScheduledForErasureSince() {
+      return this.utcDateScheduledForErasureSince;
+    }
```

Comment:
Instead of adding getters I usually make the field public. Since this pattern is already everywhere, I would rather stick with it. 

Otherwise there's always the possibility of implementing getters using the _get_ keyword.

Reply:
yeah, I wasn't sure about that – because in theory it makes sense to not be able to modify that value, but then again – we don't seem to be using that feature very consistently.

Regarding `get` – I know about that, but I wouldn't be able to name the getter the same as the field, I think? 
e.g. if the field is called `ownerId` I don't think I could have a getter called `ownerId`, but would need to rename it, e.g. like getOwnerId – and `get` to me always is reserved for some sort of method or function, so using `get getOwnerId` would feel weird, when you would call it with `class.getOwnerId` instead of `class.getOwnderId()`

but anyways – bit of useless rambling :-D

I'll make the fields public :-)